src/lib/services/types/proposal.ts

```diff
@@ -363,15 +230,51 @@ const zProposalAnswerCountsResponse = z
     }),
   })
   .transform(snakeToCamel);
-
 export type ProposalAnswerCountsResponse = z.infer<
   typeof zProposalAnswerCountsResponse
 >;
 
-export const getProposalAnswerCounts = async (
-  endpoint: string,
-  id: number
-): Promise<ProposalAnswerCountsResponse> =>
-  axios
-    .get(`${endpoint}/${encodeURIComponent(id)}/answer-counts`)
-    .then(({ data }) => parseWithError(zProposalAnswerCountsResponse, data));
+export const zProposalsResponseItemLcd = z
+  .object({
+    id: z.string(),
+    messages: z
+      .object({
+        "@type": z.string(),
+      })
+      .passthrough()
+      .array()
+      .nullable(),
+    status: zProposalStatusLcd,
+    final_tally_result: z.object({
+      yes_count: z.string(),
+      no_count: z.string(),
+      abstain_count: z.string(),
+      no_with_veto_count: z.string(),
+    }),
+    submit_time: zUtcDate,
+    deposit_end_time: zUtcDate,
+    total_deposit: zCoin.array(),
+    voting_start_time: zUtcDate,
+    voting_end_time: zUtcDate,
+    metadata: z.string(),
+    title: z.string(),
+    summary: z.string(),
+    proposer: zBechAddr,
+  })
+  .transform<Proposal>((val) => ({
+    ...snakeToCamel(val),
+    id: Number(val.id),
+    status: mapProposalStatusLcdToProposalStatus(val.status),
+    resolvedHeight: null,
+    types: [],
+    isExpedited: false,
```

Comment:
Let's derive the value of `is_expedited` and use a default of false if it remains unset.